.github/workflows/packaging.yml

```diff
@@ -0,0 +1,31 @@
+name: packaging
+
+on:
+  push:
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'fedora', dist: '33' }
+
+    env:
+      OS: ${{ matrix.platform.os }}
+      DIST: ${{ matrix.platform.dist }}
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
```

Comment:
Use `fetch-depth: 0` for correct `git describe` (and so correct package version). BTW, you can use newer checkout action version.

Reply:
Also take care to tags deannotation: add `git fetch --tags -f`.

See the following links for details:

* https://github.com/tarantool/tarantool/blob/15b823f0ee3eb142c36564fa3973993c1ceba5b9/.github/actions/environment/action.yml#L52-L74
* https://github.com/actions/checkout/issues/290

Reply:
Seems resolved.

Reply:
yep, thanks for your help